module/dice/simplify-roll-formula.mjs

```diff
@@ -150,17 +150,19 @@ function _simplifyDiceTerms(terms) {
   const diceQuantities = unannotated.reduce((obj, curr, i) => {
     if ( curr instanceof OperatorTerm ) return obj;
     const face = curr.constructor?.name === "Coin" ? "c" : curr.faces;
-    const key = `${unannotated[i - 1].operator}${face}`;
-    obj[key] = (obj[key] ?? 0) + curr.number;
+    const key = `${unannotated[i - 1].operator}${face}${curr.modifiers.filterJoin("")}`;
+    obj[key] ??= {};
+    obj[key].number = (obj[key].number ?? 0) + curr.number;
+    obj[key].modifiers = (obj[key].modifiers ?? []).concat(curr.modifiers);
     return obj;
   }, {});
 
   // Add new die and operator terms to simplified for each die size and sign
-  const simplified = Object.entries(diceQuantities).flatMap(([key, number]) => ([
+  const simplified = Object.entries(diceQuantities).flatMap(([key, {number, modifiers}]) => ([
     new OperatorTerm({ operator: key.charAt(0) }),
     key.slice(1) === "c"
```

Comment:
If the `key` now contains the modifiers, I think this test will not correctly detect a `Coin` term with modifiers.

Reply:
Counting successes is probably a very common modifier for coin flips, so I'm not sure if it's *the* solution necessary. But certainly nothing in official content that I know of, and modifiers for coin flips were ignored before, so we haven't made anything *worse* here I suppose.